scripts/generate_and_push_perf_report.sh

```diff
@@ -6,11 +6,14 @@ set -eux -o pipefail
 
 SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
-git clone https://$VIP_VAP_ACCESS_TOKEN@github.com/zenithdb/zenith-perf-data.git
+git clone --single-branch --branch testing https://$VIP_VAP_ACCESS_TOKEN@github.com/zenithdb/zenith-perf-data.git  # TODO (PRIOR MERGE) replace back before merge
 cd zenith-perf-data
 mkdir -p reports/
 mkdir -p data/$REPORT_TO
 
+# ingest per test results data into zenith backed postgres running in staging to build grafana reports on that data
+DATABASE_URL=$PERF_TEST_RESULT_CONNSTR poetry run python $SCRIPT_DIR/ingest_perf_test_result.py --ingest $REPORT_FROM
+
 cp $REPORT_FROM/* data/$REPORT_TO
```

Comment:
Friendly reminder: we could reuse `scripts/git-upload` here, but I won't mind leaving this as-is.

Reply:
Git upload needs some modifications then, html report is generated over all data in the repo, so in that case I have to inject report generation command inside git upload. Something like `git-upload copy ... --run-cmd='<update report>'`. Is it ok to have that in git upload?

Reply:
> Is it ok to have that in git upload?

I won't mind.